docs/source/errors.md

```diff
@@ -246,3 +246,9 @@ Some build errors are part of normal workflows. For instance, you may need to pu
 
 This error occurs when an operation check fails. This means that you proposed a schema that would break operations in use by existing clients. You can configure this behavior in the Checks -> Configuration view in [Apollo Studio](https://studio.apollographql.com/), and you can read more about client checks [here](https://www.apollographql.com/docs/studio/schema-checks/).
 
+### E031
+
+This error occurs when Rover made an HTTP(S) request and it timed out.
```

Comment:
"made an HTTP request that timed out." <- I'm not too stressed about needing to include the (S)